tests/base.py

```diff
@@ -86,6 +91,18 @@ def stopServer():
     """
     cherrypy.engine.exit()
     mockSmtp.stop()
+    dropAllTestDatabases()
+
+
+def dropAllTestDatabases():
+    """
+    Unless otherwise requested, drop all test databases.
+    """
+    if 'keepdb' not in os.environ.get('EXTRADEBUG', '').split():
```

Comment:
It needs to be callable from tests that don't start the server in the ordinary manner (like the sftp test).  We already have a function `dropTestDatabase` for dropping one database which can also reload models if appropriate.